test/test_for_precompile.jl

```diff
@@ -0,0 +1,17 @@
+using MakieGallery, AbstractPlotting, GLMakie, Test
+
+empty!(MakieGallery.plotting_backends)
+push!(MakieGallery.plotting_backends, "Makie")
+database = MakieGallery.load_database([
+                    "tutorials.jl",
+                    "attributes.jl",
+                    "short_tests.jl"
+                    ])
+
+tested_diff_path = joinpath(@__DIR__, "tested_different")
```

Comment:
I've no idea how many of these lines of code we actually need

Reply:
You can remove the tested diff path, since there's no comparison running

Reply:
```suggestion